### PR TITLE
181 crud operations for company entity

### PR DESCRIPTION
**Pull request by aironhight**

### CODE REVIEW

File: apps/api/src/company/dto/create-company.dto.ts [hunk 1]
```diff
@@ -12,25 +13,39 @@ export class CreateCompanyDto {
 
   @Expose()
   @IsString()
-  @ApiProperty()
+  @ApiProperty({
+    description:
```
Comment (kachar) [inline apps/api/src/company/dto/create-company.dto.ts:17]: It's awesome that you've provided a nice description to the property

File: apps/api/src/company/dto/create-company.dto.ts [hunk 2]
```diff
@@ -12,25 +13,39 @@ export class CreateCompanyDto {
...
+      'BULSTAT Unified Identification Code (UIC) https://psc.egov.bg/en/psc-starting-a-business-bulstat',
+  })
+  @IsUUID()
```
Comment (kachar) [inline apps/api/src/company/dto/create-company.dto.ts:20]: I'm not sure that the `companyNumber` is `UUID` format, maybe a typo

Comment (aironhight) [reply apps/api/src/company/dto/create-company.dto.ts:20]: Nope, it is not. It's unique in the DB, but not in UUID format.

**Review by kachar — APPROVED** [review]
Thanks for your contribution! :boat:

**Pull request merged**